src/Microsoft.Azure.WebJobs.Host.Rpc/WebJobs.Host.Rpc.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\build\common.props" />
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
This is intended only for out-of-proc worker scenarios. The functions host is `net6.0` in these cases, but do I want to multi-target this also for extensions not yet moved to `net6.0`? For example, Durable Functions host extension only targets `netcoreapp3.1` at the latest.

Should this target `netcoreapp3.1` also, or should extensions be required to upgrade considering `net6.0` is the oldest still in supported TFM (excluding netfx)?

Reply:
We can start here with `net6.0`. We'd need to make sure bundles are frozen on V3 to make sure we don't pick up those updates there, but unless we have a very good reason to expand, we should keep this scoped to `net6.0`